src/gamecontroller.cs

```diff
@@ -115,10 +123,16 @@ partial interface GCControllerDirectionPad {
 
 		[Export ("right")]
 		GCControllerButtonInput Right { get; }
+
+		[Export ("setValueForXAxis:yAxis:")]
+		void SetValueForXAxis (float xAxis, float yAxis);
```

Comment:
Just `SetValue` is enough here.

Reply:
yes, an ObjC API with two parameters looks like `methodParam1:Param2:`
IOW there's **no** `:` between the method name and the first parameter, so
* method -> `SetValueFor`
* parameter 1 -> `xAxis`
* parameter 2 -> `yAxis`

and we remove suffix like `For` (and `With`...) so it becomes 
```csharp
void SetValue (float xAxis, float yAxis);
```

Reply:
Thanks, will change!